api/http/server.go

```diff
@@ -865,6 +875,57 @@ func (s *Server) HandleGetFile(w http.ResponseWriter, r *http.Request) {
 	http.ServeContent(w, r, fileName, time.Now(), newBufferedReadSeeker(reader, getFileBufferSize))
 }
 
+// HandleGetTag responds to the following request
+//    - bzz-tag:/<manifest>  and
+//    - bzz-tag:/?tagId=<tagId>
+// Clients should use root hash or the tagID to get the tag counters
+func (s *Server) HandleGetTag(w http.ResponseWriter, r *http.Request) {
+	getTagCount.Inc(1)
+	uri := GetURI(r.Context())
+	fileAddr := uri.Address()
```

Comment:
`uri` could be nil, thus you are risking a panic here. you won't see it though, since the http server catches panics (but logs them)

Reply:
I have added it. 

Note: This is like this in all other functions.. May be we should add that too.